File: x11-wm/pager/files/patch-pager.jl
--- pager.jl.orig	Tue Nov 19 21:06:10 2002
+++ pager.jl	Tue Nov 19 21:08:30 2002
@@ -277,7 +277,10 @@
   ;; Tells the C program to change the colors
   (define (send-colors)
     (send ?c
-	  (mapcar color-rgb
+	  (mapcar (lambda (color)
+                     (let ((clist (color-rgb color)))
+                        (setcdr (nthcdr 2 clist) nil)
+                        clist))
 		  (list pager-color-background
 			pager-color-viewport
 			pager-color-window

File: x11-wm/pager/files/patch-pager.c
--- pager.c.orig	Thu Nov 21 12:46:28 2002
+++ pager.c	Thu Nov 21 12:46:48 2002
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <unistd.h>
+#include <sys/time.h>
 #include <sys/select.h>
 #include <gtk/gtk.h>
 
